src/znc.cpp

```diff
@@ -229,6 +229,15 @@ void CZNC::Loop() {
                 // stop pending configuration timer
                 DisableConfigTimer();
 
+                if (GetReadonlyConfig()) {
+                    if (eState == ECONFIG_NEED_VERBOSE_WRITE) {
+                        Broadcast("Writing the config skipped because "
```

Comment:
This will regularly spam admins whenever e.g. "in-config" channel's key is changing...

Reply:
I thought these changes trigger non-verbose config saves... I grepped for `ECONFIG_NEED_VERBOSE_WRITE` and the only place that sets it that I found was `SIGUSR1` handler.